### PR TITLE
New signature of `createElement` is introduced

### PR DESCRIPTION
**Pull request by sgrishchenko**
New signature of `createElement` is introduced, deprecation for old implementation is added.

cc @turansky @aerialist7

### CODE REVIEW

File: kotlin-react/src/main/kotlin/react/ReactElement.ext.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package react
+
+import kotlinext.js.jso
+
+inline fun createElement(block: Render): ReactElement? {
```
Comment (turansky) [inline kotlin-react/src/main/kotlin/react/ReactElement.ext.kt:5]: inline?

Comment (sgrishchenko) [reply kotlin-react/src/main/kotlin/react/ReactElement.ext.kt]: copy-paste :stuck_out_tongue_winking_eye:, fixed

File: kotlin-react/src/main/kotlin/react/ReactElement.ext.kt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package react
+
+import kotlinext.js.jso
+
+fun createElement(block: Render): ReactElement? {
```
Comment (turansky) [inline kotlin-react/src/main/kotlin/react/ReactElement.ext.kt:5]: Discussion required

File: kotlin-react/src/main/kotlin/react/ReactElement.ext.kt [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    val nodes = RBuilder().apply(block).childList
+
+    return when {
+        nodes.size == 0 -> null
+        nodes.size == 1 && isValidElement(nodes.first()) -> nodes.first().unsafeCast<ReactElement>()
+        else -> createElement(Fragment, jso(), *nodes.toTypedArray())
```
Comment (turansky) [inline kotlin-react/src/main/kotlin/react/ReactElement.ext.kt:11]: Named parameter?

**Comment by Leonya** [comment]
Released in pre.252

File: kotlin-react/src/main/kotlin/react/RBuilder.kt [hunk 4]
```diff
@@ -140,6 +140,7 @@ open class RBuilderImpl : RBuilder {
 
 open class RBuilderMultiple : RBuilderImpl()
 
+@Deprecated(message = "Unsafe API, use createElement")
```
Comment (Leonya) [inline kotlin-react/src/main/kotlin/react/RBuilder.kt:143]: How am I supposed to use it if `createElement` doesn't have the `builder` parameter?

**Pull request merged**